### PR TITLE
CIV-13109 Email notifications: Claimant/Defendant LR - set aside Judgment made in error

### PR DESCRIPTION
**Pull request by sjamihmcts**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CIV-13109

### Change description ###

To generate an email notification for Claimant and Defendant LR following Judgment set aside after an error

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/civil/callback/CaseEvent.java [hunk 1]
```diff
@@ -123,6 +123,7 @@ public enum CaseEvent {
     REQUEST_FOR_RECONSIDERATION(USER),
     UPDATE_HELP_WITH_FEE_NUMBER(USER),
     DECISION_ON_RECONSIDERATION_REQUEST(USER),
+    NOTIFY_SET_ASIDE_JUDGMENT(USER),
```
[the remark this answers is not in the document]
Comment (sjamihmcts) [reply src/main/java/uk/gov/hmcts/reform/civil/callback/CaseEvent.java]: Updated.

File: build.gradle [hunk 2]
```diff
@@ -364,7 +364,7 @@ configurations.all {
 }
 
 dependencies {
-  implementation 'com.github.hmcts:civil-commons:1.0.65'
+  implementation 'com.github.hmcts:civil-commons:v1.0.70-CIV-13109'
```
Comment (bhagyashreesharma90) [inline build.gradle:367]: adding comment so we don't merge this. we need to create release tag and merge with that

**Review by bhagyashreesharma90 — DISMISSED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/utils/NotificationUtils.java [hunk 3]
```diff
@@ -156,4 +167,23 @@ public static String getRespondentLegalOrganizationName(OrganisationPolicy organ
         }
         return respondentLegalOrganizationName;
     }
+
+    public static String getApplicantLegalOrganizationName(CaseData caseData, OrganisationService organisationService) {
```
[the remark this answers is not in the document]
Comment (sjamihmcts) [reply src/main/java/uk/gov/hmcts/reform/civil/utils/NotificationUtils.java:171]: We both have the same code in NotificationUtils class

**Review by Omaira-Melo-Hmcts — APPROVED** [review]

File: config/owasp/suppressions.xml [hunk 4]
```diff
@@ -39,6 +39,7 @@
     <cve>CVE-2023-46589</cve>
     <cve>CVE-2023-6378</cve>
     <cve>CVE-2023-34042</cve>
+    <cve>CVE-2024-1597</cve>
```
Comment (sjamihmcts) [inline config/owasp/suppressions.xml:42]: Temporary fix for build to progress. To be re-visited.

**Review by bhagyashreesharma90 — APPROVED** [review]

**Review by hmcts-version1-ignacio — DISMISSED** [review]

**Review by hmcts-version1-ignacio — DISMISSED** [review]

**Review by Azam-Hmcts — APPROVED** [review]

**Pull request merged**